File: CS/WindowsApplication1/Data/DetailItemClass.cs
using System;

namespace WindowsApplication1 {
    public class DetailItemClass {
        public DetailItemClass(TimeSpan val1, int val2, int pID) {
            this.ParentID = pID;
            this.TimeStamp = val1;
            this.Value = val2;
        }
        public int ParentID { get; set; }
        public TimeSpan TimeStamp { get; set; }
        public int Value { get; set; }

    }
}

File: VB/WindowsApplication1/Data/DetailItemClass.vb
Imports System

Namespace WindowsApplication1
    Public Class DetailItemClass
        Public Sub New(ByVal val1 As TimeSpan, ByVal val2 As Integer, ByVal pID As Integer)
            Me.ParentID = pID
            Me.TimeStamp = val1
            Me.Value = val2
        End Sub

        Public Property ParentID As Integer
        Public Property TimeStamp As TimeSpan
        Public Property Value As Integer
    End Class
End Namespace
